### PR TITLE
Itsology

### PR DESCRIPTION
**Pull request by smurfix**
Fix a couple of questionable (and one 100% wrong) instances of "it's".

### CODE REVIEW

File: docs/source/quickstart.rst [hunk 1]
```diff
@@ -288,7 +288,7 @@ server. Sometimes this will work fine. But it takes a little while for
 the server to start up and be ready to accept connections – so other
 times, randomly, our connection attempt will happen too quickly, and
 error out. After all – ``nursery.start_soon`` only promises that the
-task will be started *soon*, not that it's actually happened. So this
+task will be started *soon*, not that it has actually happened. So this
```
Comment (belm0) [inline docs/source/quickstart.rst]: "it's" as a contraction for "it has" is valid and common

File: pytest_trio/plugin.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ def pytest_exception_interact(node, call, report):
 #
 # You should think of fixtures as a dependency graph: each fixtures *uses*
 # zero or more other fixtures, and is *used by* zero or more other fixtures. A
-# fixture should be setup before any of the fixtures it's used by are setup,
+# fixture should be setup before any of the fixtures it is used by are setup,
```
Comment (belm0) [inline pytest_trio/plugin.py]: in this case "it's" seems to help the readability of this awkward sentence

perhaps "A fixture should be setup before any of its dependees are setup, and torn down once all of its dependees are terminated."

File: docs/source/quickstart.rst (hunk 1 above)
Comment (smurfix) [reply docs/source/quickstart.rst]: Yes it is common, but the contracion's validity depends on the emphasis being on the "it" part. In this sentence however the emphasis is on the "has", thus the contraction isn't approriate.

File: pytest_trio/plugin.py (hunk 2 above)
Comment (smurfix) [reply pytest_trio/plugin.py]: Good alternate version, I'll take it ;-)

**Comment by smurfix** [comment]
Thanks for your ACK.

**Pull request merged**